integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableRenameCommand.scala

```diff
@@ -138,6 +147,27 @@ private[sql] case class CarbonAlterTableRenameCommand(
           sys.error(s"Folder rename failed for table $oldDatabaseName.$oldTableName")
         }
       }
+      val updatedParts = updatePartitionLocations(
```

Comment:
need to check once how hive managed table rename works? whether it renames location or not? If not how table creation with old table name works?

Reply:
ok